charts/oes/values.yaml

```diff
@@ -706,7 +706,7 @@ spinnaker:
         front50-local.yml:
           policy:
             opa:
-              enabled: true
+              enabled: false
```

Comment:
If we leave it enabled, would it create a problem?

Reply:
Sample pipelines are not getting saved when opa is enabled